code/web/src/modules/user/EditProfile.js

```diff
@@ -53,84 +54,90 @@ class EditProfile extends Component {
-  //       this.setState({
-  //         isLoading: false,
-  //       })
+    //call to back-end to post/update new data (method below does not exist yet)
+    this.props.updateProfileInfo(this.state.newProfileData)
+      .then((response) => {
+        this.setState({
+          isLoading: false,
+        })
         
-  //       if (response.data.errors && response.data.errors.length > 0) {
-  //         this.props.messageShow(response.data.errors[0].message)
-  //       } else {
-  //         this.props.messageShow('Information saved successfully.')
-  //         //might need something else here
-  //         //might need to save image path on user
-  //       }
-  //     })
-
-  //     .catch((error) => {
-  //       this.props.messageShow('There was some error. Please try again.')
-
-  //       this.setState({
-  //         isLoading: false,
-  //       })
-  //     })
-  //     .then(() => {
-  //       window.setTimeout(() => {
-  //         this.props.messageHide()
-  //       }, 5000)
-  //     })
-  // }
-
-  // onUpload = (event) => {
-  //   this.props.messageShow('Uploading photo, please wait...')
-
-  //   this.setState({
-  //     isLoading: true,
-  //   })
-
-  //   let data = new FormData()
-  //   data.append('file', event.target.files[0])
-
-  //   // Upload image
-  //   this.props.upload(data)
-  //     .then((response) => {
-  //       if (response.status === 200) {
-  //         this.props.messageShow('File uploaded successfully.')
-
-  //         let image = this.state.image
-  //         image = `/images/uploads/${response.data.file}`
-
-  //         this.setState({
-  //           image,
-  //         })
-  //       } else {
-  //         this.props.messageShow('Please try again.')
-  //       }
-  //     })
-  //     .catch((error) => {
-  //       this.props.messageShow('There was some error. Please try again.')
-  //     })
-  //     .then(() => {
-  //       this.setState({
-  //         isLoading: false,
-  //       })
-
-  //       window.setTimeout(() => {
-  //         this.props.messageHide()
-  //       }, 5000)
-  //     })
-  // }
+        if (this.props.user.error !== '') {
+          this.props.messageShow(this.props.user.error)
+        } else {
+          this.props.messageShow('Information saved successfully.')
+          //might need something else here
+          //might need to save image path on user
+        }
+      })
+      .catch((error) => {
+        this.props.messageShow('There was some error. Please try again.')
+
+        this.setState({
+          isLoading: false,
+        })
+      })
+      .then(() => {
+        window.setTimeout(() => {
+          this.props.messageHide()
+        }, 5000)
+      })
+  }
+
+  onUpload = (event) => {
+    this.props.messageShow('Uploading photo, please wait...')
+
+    this.setState({
+      isLoading: true,
+    })
+
+    let data = new FormData()
+    data.append('file', event.target.files[0])
+
+    // Upload image
+    this.props.upload(data)
```

Comment:
I'm curious about industry practices with uploading and storing images (and other files types)